cocoa/src/toga_cocoa/fonts.py

```diff
@@ -55,11 +55,13 @@ def __init__(self, interface):
                 else:
                     raise ValueError(f"Font file {font_path} could not be found")
 
-            # Default system font size on Cocoa is 12pt
             if self.interface.size == SYSTEM_DEFAULT_FONT_SIZE:
                 font_size = NSFont.systemFontSize
             else:
-                font_size = self.interface.size
+                # A "point" in Apple APIs is equivalent to a CSS pixel, but the Toga
+                # public API works in CSS points, which are slightly larger
+                # (https://developer.apple.com/library/archive/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Explained/Explained.html).
+                font_size = self.interface.size * 96 / 72
```

Comment:
This interpretation of font size makes sense - but it seems like something we should capture on the Font documentation page, along with platform notes for each platform that provides specifics for the benefit of anyone who might have platform experience.

Those details should then be cross-linked to the Pack style page discussing `font-size` attribute.

Reply:
Done.